vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/DeviceSteps.java

```diff
@@ -76,21 +80,68 @@ public void uploadFileToDevice(String filePath)
      * See <a href="https://github.com/appium/appium-xcuitest-driver#mobile-pressbutton">iOS keys</a> and
      * <a href="https://appium.github.io/java-client/io/appium/java_client/android/nativekey/AndroidKey.html">
      * Android keys</a> for available values
+     * <br>
+     * Example:
+     * <br>
+     * <code>
+     * When I press $key key
+     * </code>
      *
-     * @param key key to press
+     * @param key the key to press
      */
     @When("I press $key key")
     public void pressKey(String key)
+    {
+        pressKeys(List.of(key));
+    }
+
+    /**
+     * Presses the keys
+     * <br>
+     * See <a href="https://github.com/appium/appium-xcuitest-driver#mobile-pressbutton">iOS keys</a> and
+     * <a href="https://appium.github.io/java-client/io/appium/java_client/android/nativekey/AndroidKey.html">
+     * Android keys</a> for available values
+     * <br>
+     * Example:
+     * <br>
+     * <code>
+     * When I press keys:
+     * <br>
+     * |key |
+     * <br>
+     * |Home|
+     * </code>
+     *
+     * @param keys the keys to press
+     */
+    @When("I press keys:$keys")
+    public void pressKeys(ExamplesTable keys)
+    {
+        pressKeys(MapUtils.convertExamplesTableToMap(keys).get("key"));
+    }
+
+    private void pressKeys(List<String> keys)
     {
         if (genericWebDriverManager.isIOSNativeApp())
         {
-            javascriptActions.executeScript("mobile: pressButton", Map.of("name", key));
+            keys.forEach(key -> javascriptActions.executeScript("mobile: pressButton", Map.of("name", key)));
         }
         else
         {
-            AndroidKey androidKey = EnumUtils.getEnumIgnoreCase(AndroidKey.class, key);
-            Validate.isTrue(androidKey != null, "Unsupported Android key: %s", key);
-            webDriverProvider.getUnwrapped(PressesKey.class).pressKey(new KeyEvent(androidKey));
+            keys.stream()
+                .map(key ->
+                {
+                    AndroidKey androidKey = EnumUtils.getEnumIgnoreCase(AndroidKey.class, key);
+                    Validate.isTrue(androidKey != null, "Unsupported Android key: %s", key);
+                    return androidKey;
+                })
+                .map(KeyEvent::new)
+                .collect(Collectors.collectingAndThen(Collectors.toList(), keyEvents ->
+                {
+                    PressesKey pressesKey = webDriverProvider.getUnwrapped(PressesKey.class);
+                    keyEvents.forEach(pressesKey::pressKey);
```

Comment:
it could be a part of the wrapping stream

Reply:
it needs to eliminate multiple invocations of webDriverProvider.getUnwrapped(PressesKey.class)

Reply:
just unwrap it once before stream start

Reply:
what if one of the keys is invalid? then the wrapping becomes pointless

Reply:
tricky question: what is better to create a list or to cast an object to another type